### PR TITLE
Fix issues during CV review

### PR DESCRIPTION
**Pull request by junrongYu**
Issues listed here:
https://docs.google.com/spreadsheets/d/1h8lZagQQE3LSIAs1tA8lDBmgyw1QJfOn_X1-tjU2XMw/edit#gid=0

### CODE REVIEW

**Review by smguru — CHANGES_REQUESTED** [review]
The definition should be 
"The dimension of the equipment used during a passive-targeted fauna survey."

**Review by smguru — CHANGES_REQUESTED** [review]
the definition should be "The dimensions of the equipment used during a passive-targeted fauna survey."

File: vocab_files/attribute_concepts/transect-start-point.ttl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schema: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/9beaf2dc-c53d-4e6e-b699-c1786f04f1f6>
+    a skos:Concept ;
+    dcterms:source """McCallum K, Laws M, Bignall J, O’Neill S, Sparrow B. (unpublished draft) ‘Coarse Woody Debris module’ in ….. (eds)
```
Comment (smguru) [inline vocab_files/attribute_concepts/transect-start-point.ttl:10]: there are more "s in the souce

File: vocab_files/attribute_concepts/equipment-dimensions.ttl [hunk 2]
```diff
@@ -11,9 +11,9 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision
 making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "The dimensions of the equiments used during a pasive- targeted fauna survey. " ;
+    skos:definition "The dimensions of the equipments used during a passive- targeted fauna survey." ;
```
Comment (smguru) [inline vocab_files/attribute_concepts/equipment-dimensions.ttl:14]: the definition should be "The dimension of the equipment used during a passive-targeted fauna survey."

File: vocab_files/attribute_concepts/soil-bulk-density-location.ttl [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/1d7974b3-892b-47d5-8fdb-32f36f331e05>
+    a skos:Concept ;
+    dcterms:source """Laws M, McCallum K, Bignall J, O’Neill S, Sparrow B. (unpublished draft) ‘Soils module’ in ….. (eds) Ecological Field
+Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision
+making. Draft v 0.1 Report to DAWE. TERN, Adelaide""" ;
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:definition "The sampling location/s on the plot where soil samples were collected for soil bulk density analysis." ;
+    skos:prefLabel "soil bulk density location" ;
```
Comment (smguru) [inline vocab_files/attribute_concepts/soil-bulk-density-location.ttl:15]: do we really need this concept, I am sure there is a "soil sample location"

File: vocab_files/attribute_concepts/bird-survey-location.ttl [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/78ae22e5-ccf3-4b2c-b07f-1f1120da8d3f>
+    a skos:Concept ;
+    dcterms:source """O’Neill S, McCallum K, Bignall J, Laws M, and Sparrow B, (unpublished draft) ‘Vertebrate Fauna module’ in ….. (eds)
+Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved
+decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:definition "The location where the bird survey weas conducted. Options for 'within survey plot', 'outside survey plot', 'same habitat', 'outside plot, different habitat'" ;
```
Comment (smguru) [inline vocab_files/attribute_concepts/bird-survey-location.ttl:14]: Shouldn't this be a categorical variable or represented as a concept collection and define each value separately?

File: vocab_files/observable_property_concepts/redeployment-observations.ttl [hunk 5]
```diff
@@ -2,16 +2,14 @@ PREFIX dcterms: <http://purl.org/dc/terms/>
...
 PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
-PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/nrm/0adc5740-7e20-4509-949c-12ae72500951>
     a skos:Concept ;
     dcterms:source "Laws M, McCallum K, Bignall J, O’Neill S, Sparrow B. (unpublished draft) ‘Camera Traps Module’ in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "A brief summary of the observations made while redployment of the camera traps in the study site. " ;
+    skos:definition "A brief summary of the observations made while redeployment of the camera traps in the study site. " ;
```
Comment (smguru) [inline vocab_files/observable_property_concepts/redeployment-observations.ttl:11]: The definition should be "A brief summary of observations made during the redeployment of camera traps in the study site. "

File: vocab_files/observable_property_concepts/mean-basal-area.ttl [hunk 6]
```diff
@@ -6,7 +6,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://linked.data.gov.au/def/nrm/a70282a3-c62f-43df-8d9c-34727ae1ee16>
     a skos:Concept ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "TBD" ;
+    skos:definition "The average basal area for a given plot, expressed in square meters (m^2) per unit area of the plot." ;
```
Comment (smguru) [inline vocab_files/observable_property_concepts/mean-basal-area.ttl:9]: the definition should be "The average amount of area for a given plot occupied by tree stems, expressed in square meters (m^2) per unit area of the plot." ; I am not sure whether this is correct,  I don't think they do average; it is just a total cross-sectional area of all stems  measured and expressed as sq mt/hectare. probably good to confirm

File: vocab_files/attribute_concepts/voucher-condition.ttl [hunk 7]
```diff
@@ -6,11 +7,12 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/nrm/010ee3c8-bfbe-47c5-adc8-068ca664173d>
     a skos:Concept ;
+    dcterms:source "O’Neill S, McCallum K, Bignall J, Laws M, and Sparrow B, (unpublished draft) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "The condition of the voucher. E.g. is 'Fresh'." ;
+    skos:definition "The condition of the voucher specimen/s collected during an ecological survey. E.g. is 'Fresh'." ;
```
Comment (smguru) [inline vocab_files/attribute_concepts/voucher-condition.ttl:12]: are there any predefined values?

File: vocab_files/attribute_concepts/survey-design-and-justification.ttl [hunk 8]
```diff
@@ -9,7 +9,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     a skos:Concept ;
     dcterms:source "McCallum K, Bignall J, Laws M, O’Neill S, Sparrow B. (unpublished draft) ‘Targeted Surveys module’ in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "A free text summary of the targeted survey design and justification for conducting the same." ;
+    skos:definition "The survey deign and justification includes any details on the field survey methods being implemented (i.e. combination of methods used), what the survey design is based on (e.g. EPBC survey guidelines, scientific paper) and what is being targeted." ;
```
Comment (smguru) [inline vocab_files/attribute_concepts/survey-design-and-justification.ttl:12]: "The survey design and justification include any details on the field survey methods being implemented (i.e. combination of methods used), what the survey design is based on (e.g. EPBC survey guidelines, scientific paper) and what is being targeted."

Comment (smguru) [reply vocab_files/attribute_concepts/survey-design-and-justification.ttl:12]: I have fixed few typos

**Review by smguru — CHANGES_REQUESTED** [review]
@junrongYu  I have added some comments and some updates to the definitions.

File: vocab_files/observable_property_concepts/mean-basal-area.ttl (hunk 6 above)
Comment (junrongYu) [reply vocab_files/observable_property_concepts/mean-basal-area.ttl:9]: probably it's added in the app and dataset, from the protocol:
_Repeat steps 3–11 at each remaining sampling location. When all the sampling locations have been assessed, the App will display a data summary (e.g. mean basal area, total stem counts) on the sampling location selection screen._

File: vocab_files/attribute_concepts/bird-survey-location.ttl (hunk 4 above)
Comment (junrongYu) [reply vocab_files/attribute_concepts/bird-survey-location.ttl:14]: yes it is categorical with the value type `tern:IRI` and the categorical values collection is https://linked.data.gov.au/def/nrm/2f0470aa-2c05-4e87-a5ac-1d50205b9927

File: vocab_files/attribute_concepts/soil-bulk-density-location.ttl (hunk 3 above)
Comment (junrongYu) [reply vocab_files/attribute_concepts/soil-bulk-density-location.ttl:15]: we have an attribute `soil sampling location description`

File: vocab_files/attribute_concepts/transect-start-point.ttl (hunk 1 above)
Comment (junrongYu) [reply vocab_files/attribute_concepts/transect-start-point.ttl:10]: three quotes are used when the text is more than one line

File: vocab_files/attribute_concepts/voucher-condition.ttl (hunk 7 above)
Comment (junrongYu) [reply vocab_files/attribute_concepts/voucher-condition.ttl:12]: yes, categorical values for `voucher condition` is in this collection https://linked.data.gov.au/def/nrm/f0647b85-a014-4299-b23f-6aeb25e834bc

**Review by smguru — APPROVED** [review]

**Pull request merged**